.github/workflows/smi-conformance.yml

```diff
@@ -0,0 +1,49 @@
+
+name: service-mesh-performance
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  workflow_dispatch:
+
+jobs:
+  perftest:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install mesheryctl
+        run: |
+          /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
+          eval "$(/home/linuxbrew/.linuxbrew/bin/brew shellenv)"
+          brew tap layer5io/tap
+          brew install mesheryctl
+          
+      - name: export token
+        run: export TOKEN="${{ secrets.AUTH_TOKEN }}"
+
+      - name: pipe to file
+        run: |
+          echo '{ "meshery-provider": "Meshery", "token": null }' | jq '.token = env.TOKEN' > ~/auth.json
+
+      - name: setup minikube
+        uses: manusa/actions-setup-minikube@v2.4.1
+        with:
+          minikube version: 'v1.16.0'
+          kubernetes version: 'v1.19.2'
+
+      - name: deploy meshery
+        run: |
+          mesheryctl system context create k8s -p kubernetes -s
+          mesheryctl system start --yes
+
+
+      - name:  configure meshery
+        run: |
+          mesheryctl system config minikube --token ~/auth.json
+          minikube tunnel /dev/null 2>&1
+
+      - name: perfom smi conformance tests
+        run: mesheryctl perf --name "inital perf tests" --url https://docs.meshery.io --qps 300 --concurrent-requests 2 --duration 30s --token ~/auth.json
```

Comment:
When you're ready, we'll want to have these parameters as workflow input via a `-f` flag (an SMP compatible file containing the performance profile).

Reply:
Will you use a different target and lower the intensity, so that we don't trip a red flag hammering on GitHub Pages?

Reply:
```suggestion
        run: mesheryctl perf --name "inital perf tests" --url https://www.google.com --qps 5 --concurrent-requests 1 --duration 10s --token ~/auth.json
```

Reply:
Haha! Your recent joke became reality. :)